modules/core/arrow-core-data/src/test/kotlin/arrow/core/IorTest.kt

```diff
@@ -44,7 +48,9 @@ class IorTest : UnitSpec() {
       ShowLaws.laws(Ior.show(), EQ) { Right(it) },
       MonadLaws.laws(Ior.monad(Int.semigroup()), Eq.any()),
       TraverseLaws.laws(Ior.traverse(), Ior.applicative(Int.semigroup()), ::Right, Eq.any()),
-      HashLaws.laws(Ior.hash(Hash.any(), Int.hash()), Ior.eq(Eq.any(), Int.eq())) { Right(it) }
+      HashLaws.laws(Ior.hash(Hash.any(), Int.hash()), Ior.eq(Eq.any(), Int.eq())) { Right(it) },
+      BifoldableLaws.laws(Ior.bifoldable(), { Right(it) }, Eq.any()),
+      BitraverseLaws.laws(Ior.bitraverse(), { Right(it) }, Eq.any())
```

Comment:
Bitraverse Laws imply that the datatype also passes the Bifoldable Laws, so these two lines are redundant.

Reply:
Make sense.